src/main/java/edu/ucsb/ucsbcslas/controllers/TutorAssignmentController.java

```diff
@@ -221,55 +235,109 @@
         String body = mapper.writeValueAsString(tutorAssignment.get());
         return ResponseEntity.ok().body(body);
     }
-  
+
     @PutMapping(value = "/api/member/tutorAssignments/{id}", produces = "application/json")
     public ResponseEntity<String> updateTutorAssignment(@RequestHeader("Authorization") String authorization,
-        @PathVariable("id") Long id, @RequestBody @Valid String incomingTutorAssignment) throws JsonProcessingException {
-
-      if (!authControllerAdvice.getIsAdmin(authorization))
-        return getUnauthorizedResponse("admin");
-      Optional<TutorAssignment> tutorAssignment = tutorAssignmentRepository.findById(id);
-      if (!tutorAssignment.isPresent()) {
-        return ResponseEntity.notFound().build();
-      }
-      logger.info(incomingTutorAssignment);
-      JSONObject ta = new JSONObject(incomingTutorAssignment);
-      TutorAssignment newAssignment = new TutorAssignment();
-      logger.info("ta: ", ta.toString());
-      JSONObject cInfo = new JSONObject(ta.get("course").toString());
-      logger.info("cInfo: ", cInfo.toString());
-      Course c = new Course(cInfo.getLong("id"), cInfo.getString("name"), cInfo.getString("quarter"), 
-        cInfo.getString("instructorFirstName"), cInfo.getString("instructorLastName"), cInfo.getString("instructorEmail"));
-      newAssignment.setCourse(c);
-
-
-      logger.info("tutorAssignment: ", tutorAssignment.get().getTutor().getEmail());
-      logger.info("ta: ", ta.getString("tutorEmail"));
-      if(!(ta.getString("tutorEmail").equals(tutorAssignment.get().getTutor().getEmail()))){
-        Optional<Tutor> tutor = tutorRepository.findByEmail(ta.getString("tutorEmail"));
-        logger.info("tutor: ", tutor);
-        if(tutor.isPresent()){
-          newAssignment.setTutor(tutor.get());
-        }
-        else{
-          return getIncorrectInputResponse();
+            @PathVariable("id") Long id, @RequestBody @Valid String incomingTutorAssignment)
+            throws JsonProcessingException {
+
+        if (!authControllerAdvice.getIsAdmin(authorization))
+            return getUnauthorizedResponse("admin");
+        Optional<TutorAssignment> tutorAssignment = tutorAssignmentRepository.findById(id);
+        if (!tutorAssignment.isPresent()) {
+            return ResponseEntity.notFound().build();
         }
-      } else {
+        logger.info(incomingTutorAssignment);
+        JSONObject ta = new JSONObject(incomingTutorAssignment);
+        TutorAssignment newAssignment = new TutorAssignment();
+        logger.info("ta: ", ta.toString());
+        JSONObject cInfo = new JSONObject(ta.get("course").toString());
+        logger.info("cInfo: ", cInfo.toString());
```

Comment:
I believe `cInfoo.toString()` is uneccessary, you can just replace with `ta` and it will implicitly call toString.

Up to you if you want to keep the `toString` invocation.

Reply:
Is the same case with this code. I wrote the one that was after the code on line 286 where it says, "Controller to upload CSV for tutorAssignment Service" 

I was a bit confused on how it got included on my PR

Reply:
I see, no worries then.